meson_options.txt

```diff
@@ -0,0 +1,16 @@
+option('MFX_DECODER', type : 'boolean', value : true)
+option('USE_HEVC_DECODER', type : 'boolean', value : true)
+option('USE_VP8_DECODER', type : 'boolean', value : true)
+option('USE_VP9_DECODER', type : 'boolean', value : true)
```

Comment:
Somehow, for this case, on my Haswell Win10 system, it failed to compile. MSS 2017 R1, somehow R2 ain't working for me.
The latter option is reasonable, given the fact if we target cross-platform support including Linux for older Haswell systems, MSS 2016 Linux is the last version that supports Haswell, that version doesn't have VP9 support.
Ideally, we should determine the codec support at runtime instead of compile time. So eventually it should be turned on, but the MFX decode plugin ranking should be changed accordingly based on the platform support detected at runtime, before registering the decoders.